timeline-chart/src/time-graph-font-controller.ts

```diff
@@ -72,4 +72,13 @@ export class FontController {
         }
         return fontName ? fontName : "";
     }
+
+    private getCharacterSet(): string[][] {
+        let letters: string[][]= [];
+        letters.push(PIXI.BitmapFont.ASCII[0]);
+        letters.push(['★','★']);
```

Comment:
we don't need the star, I suggested it as a joke to test that it works. But if we want to keep it, let's do that! :)